benchmarks/test_halomod.py

```diff
@@ -0,0 +1,46 @@
+import numpy as np
```

Comment:
There is a test_halomod and a test_halomodel. It seems one does the benchmarking and the other the unit test. It might be useful to have this clarification in the name of the file. E.g., test_halomodel_unit and test_halomodel_benchmark. This would also make it more evident to developers that they need to provide both type of tests.

Reply:
I am not following. Unit tests live with the code and benchmarks live in the benchmarks folders. Their names being similar doesn't show up when you browse the file tree. Is there something I am missing?